Buddies/Controls/Theme.swift

```diff
@@ -16,6 +16,8 @@ class Theme {
     static let themeAlt =  UIColor(red: 126/255, green: 92/255, blue: 171/255, alpha: 1.0)
     static let white = UIColor(red: 1, green: 1, blue: 1, alpha: 0.8)
     static let bad = UIColor.red
+    static let textAreaInset = UIEdgeInsets(top: 12 ,left: 7, bottom: 12, right: 7)
```

Comment:
i had made the `// Other constants` part of this for things like this, and then just had colors in this top section. obviously not a big deal whatsoever

Reply:
Nice.  I committed this before I rebased, so probably didn't see it.